code/stable/glassbr/SRS/GlassBR_SRS.tex

```diff
@@ -956,13 +956,14 @@ \subsubsection{Instance Models}
                                              \\ \midrule \\
                                              Output Constraints & \\ \midrule \\
                                                                   Equation & \begin{displaymath}
-                                                                             q=q\left({w_{TNT}},SD\right)
+                                                                             q=interpY\left(TSD.txt,SD,{w_{TNT}}\right)
```

Comment:
Strictly speaking, this highlights a but as TSD.txt is a string, and should be in string quotes. The bug is likely in the renderer somewhere, as the model contains the correct information. [I won't let this hold up the merge]